### DISCUSSION TITLE
Character sheet

### DISCUSSION THREAD

**Discussion by Cristalnoir** [Mezryss/FVTT-Genesys#118] [Q&A] [answered]
Hello, I would like to make a character sheet that will be more heroic/fantasy. I looked at the module files, but I don't see where the background of the character sheet is stored.

**Comment by Assembling-Kings** [comment, marked as answer]
This is the file that holds that styling: https://github.com/Mezryss/FVTT-Genesys/blob/main/src/actor/GenesysActorSheet.scss

**Reply by Cristalnoir** [reply]
I don't see how the code is structured, it doesn't matter. I already congratulate you for your work, and I use your module for my games. Is a French translation and different themes planned later?

**Reply by Assembling-Kings** [reply]
Thanks!

There are no plans for different themes just yet, maybe in the future when I get to implementing some of the splat books.

And regarding a French translation, @ZolOnTheNet recently published a PR with French translations. It's being merged already so it should be available on the _next_ release.

**Reply by Cristalnoir** [reply]
thanks

**Comment by Cristalnoir** [comment]
I was actually talking about the design of the character sheet. The css file must be on the hard drive, but you gave me a "github" link. Will the "Genesys" module load CSS files from the internet, rather than on the hard drive?

**Reply by Assembling-Kings** [reply]
Ahh, you wanted to make changes locally. Then you can look inside the genesys system folder, there should be only one `.css` file in there that contains everything.
